block_additional_mappings/smpte12m_timecode.md

```diff
@@ -0,0 +1,51 @@
+## Block Additional Mapping - SMTPE 12M Timecode
+
+### Timecode Description
+
+SMPTE 12 timecode values can be stored in the `BlockMore Element` to associate the content of a Matroska Block with a particular timecode value.
+
+The Block Additional Mapping contains a full binary representation of a 64 bit SMPTE timecode value expressed exactly as defined in Section 7 and 8 of SMPTE 12M [@!SMPTE.12M.2014]. For convenience, here are the bit assignments from SMPTE 12M, for information:
+
+| Bit Positions | Label                  |
+|:-------------:|:-----------------------|
+| 0--3          | Units of frames        |
+| 4--7          | First binary group     |
+| 8--9          | Tens of frames         |
+| 10            | Drop frame flag        |
+| 11            | Color frame flag       |
+| 12--15        | Second binary group    |
+| 16--19        | Units of seconds       |
+| 20--23        | Third binary group     |
+| 24--26        | Tens of seconds        |
+| 27            | Polarity correction    |
+| 28--31        | Fourth binary group    |
+| 32--35        | Units of minutes       |
+| 36--39        | Fifth binary group     |
+| 40--42        | Tens of minutes        |
+| 43            | Binary group flag BGF0 |
+| 44--47        | Sixth binary group     |
+| 48--51        | Units of hours         |
+| 52--55        | Seventh binary group   |
+| 56--57        | Tens of hours          |
+| 58            | Binary group flag BGF1 |
+| 59            | Binary group flag BGF2 |
+| 60--63        | Eighth binary group    |
+
+For example, a timecode value of `07:32:54;18` can be expressed as a 64 bit SMPTE 12M value as:
+
+```
+10000000 01100000 01100000 01010000
+00100000 00110000 01110000 00000000
+```
+
+### BlockAddIDType
+
+The BlockAddIDType value reserved for timecode is 12.
```

Comment:
I am still reluctant with small BlockAddIDType values, due to risk of conflicts.
Anyway, this value should be added in a dedicated doc which lists the used BlockAddIDType instead of having to skim the whole spec for finding all already used values.

Reply:
I'm adding a summary table of all registered BlockAddIDType values, though only one so far. Would you prefer 121 over 12?

Reply:
> Would you prefer 121 over 12?

I would :), while understanding that "12" is more well known.

Reply:
updated